charts/bulk-scan-payment-processor/values.yaml

```diff
@@ -18,6 +20,9 @@ java:
         - site-id-finrem
         - payments-queue-listen-connection-string
         - app-insights-instrumentation-key
+        - idam-client-secret
+        - idam-users-bulkscan-username
```

Comment:
I would imagine you would add idam user config for other jurisdiction later e.g probate

Reply:
Yes, I'll add other service users as the last step in this story

Reply:
On a second thought, adding them now would help avoid bumping version twice. Will do it now.